packages/common/src/audius-query/createApi.ts

```diff
@@ -296,6 +297,14 @@ const fetchData = async <Args, Data>(
 
     endpoint.onQuerySuccess?.(data, fetchArgs, { dispatch })
   } catch (e) {
+    context.reportToSentry({
+      error: e as Error,
+      level: ErrorLevel.Error,
+      additionalInfo: { fetchArgs, endpoint },
```

Comment:
ah good idea!

Reply:
Wondering if we should just restrict this to the call to `endpoint.fetch` or keep it around the whole block. This will capture errors in the before and after handlers which is probably good, and I don't think dispatching the actions should be too noisy so yeah let's go with this.

Reply:
SG!